draft-ietf-webtrans-http3.md

```diff
@@ -231,23 +242,27 @@ in {{Section 3 of RESET-STREAM-AT}}.
 ## Creating a New Session
 
 As WebTransport sessions are established over HTTP/3, they are identified using
-the `https` URI scheme ([HTTP], Section 4.2.2).
+the `https` URI scheme ({{Section 4.2.2 of HTTP}}).
 
-In order to create a new WebTransport session, a client can send an HTTP extended
-CONNECT request.  The `:protocol` pseudo-header field ({{!RFC8441}}) MUST be set to
-`webtransport`.  The `:scheme` field MUST be `https`.  Both the `:authority`
-and the `:path` value MUST be set; those fields indicate the desired
-WebTransport server.  If the WebTransport session is coming from a browser
-client, an `Origin` header {{!RFC6454}} MUST be provided within the request;
-otherwise, the header is OPTIONAL.
+In order to create a new WebTransport session, a client sends an HTTP extended
```

Comment:
Should we specify WebTransport client here or leave it ambiguous?  Or does client always mean WebTransport client?

Reply:
Client always means WebTransport client, application or application client always says "application", but anywhere it's not obvious I've been saying "WebTransport client" to be extra clear

Reply:
Updated